### ISSUE TITLE
Setting item track to play a random track plays two random tracks instead of one

### ISSUE THREAD

**Issue by maxphoenix01** [death-save/maestro#155]
**Describe the bug**
As mentioned in the title

**To Reproduce**
1. Go to 'Item track'
2. Select 'Item playlist'
3. Select 'Play random track'

4. Items play two random tracks from playlist in and out of combat

**Expected behavior**
Should be playing one random track instead of two

**Screenshots**

**Foundry Setup (please complete the following information):**
 - OS: [Windows 10]
 - Browser [Chrome]
 - Foundry Version [0.8.9]
 - Maestro Version [eg. 0.7.6])

**Additional context**

**Comment by eclarke12** [comment]
What game system is this occurring in?

**Comment by maxphoenix01** [comment]
The system is DnD 5e. And no I don't have two GMs logging in. I'm the only DM

**Comment by eclarke12** [comment]
Thus far I can't reproduce this. 
Can you please send a screenshot of the item tracks playlist, as well as a list of installed modules?

**Comment by maxphoenix01** [comment]
My modules are as follows:

Active Modules:
----------
Active-Auras v0.3.08;
Advanced Macros v1.12;
Alternative Rotation v1.3.2;
Ambient Doors v2.2.5;
Arms Reach v1.1.1;
Automated Animations v0.6.71;
Background Volume v1.4.2;
Better Curses v3.0.0;
Better Target v0.1.19;
Changelogs &amp; Conflicts v0.7.6;
CodeMirror v5.58.3-fvtt3;
Combat Enhancements v1.0.2;
Combat Utility Belt v1.7.3;
CommunityLighting by Blitz v0.4.7;
Compendium Browser v0.7.1;
Compendium Folders v2.3.64;
DFreds Droppables v1.4.0;
Dice So Nice! v4.2.2;
Dice Tray v1.3.2;
DnD5e Helpers v3.0.4;
Drag Ruler v1.9.0;
Drag Upload (Get Over Here!) v2.0.0;
Dynamic Active Effects SRD v6.0.01;
Dynamic effects using Active Effects v0.8.75;
Easy Target v2.12;
Everybody Look v2.0.0;
Forien's Quest Log v0.7.9;
Forien's Unidentified Items v0.3.17;
Foundry Community Macros v0.51.0;
Foundry Community Tables v9.1;
FoundryVTT Arms Reach v2.1.6;
FXMaster v2.0.2;
GM Notes v0.4.1;
Grid Scale Menu v0.0.14;
Initiative Double Click v1.2;
Item Containers v1.8.15;
Jack Kerouac's Animated Spell Effects v0.6.8;
JB2A - Patreon Complete Collection v0.3.3;
Jinker's Animated Art Pack v0.6;
Keybind Lib v0.3.4;
Kobold Press OGL Compendium v0.26;
Laptop Form Fix v0.0.8;
Less Fog v0.9.0;
Let's Trade 5e v0.3.0;
lib - Color Settings v2.8.1;
Lib: Document Sheet Registrar v0.6.3;
libWrapper v1.11.0.1;
Lordu's Custom Dice for Dice So Nice v0.33;
Maestro v0.7.6;
Magic Items v2.1.2;
Math.js v7.5.1-fvtt2;
Midi QOL v0.8.98;
Monk's Active Tile Triggers v1.0.38;
Monk's Enhanced Journal v1.0.35;
Monk's Little Details v1.0.39;
Monk's Scene Navigation v1.0.18;
Monk's TokenBar v1.0.55;
Monk's Wall Enhancement v1.0.5;
PDFoundry v0.8.10;
Perfect Vision v2.10.8;
Permission Viewer v0.8.14;
Pin Cushion v1.5.0;
Pings v1.2.12+0.8.6;
Plutonium v1.43.1;
Polyglot v1.7.28;
PopOut! v2.4;
Popout Resizer v1.4.1;
Quick Insert - Search Widget v2.5.3;
Quick Module Enable v1.3.4;
Search Anywhere v2.2.1;
Selective Show v1.2;
Select tool everywhere v1.2.0;
Sequencer v1.3.5;
Settings Extender v1.2.0;
Show Art v1.1.4;
Smart Doors v1.2.9;
socketlib v1.0.9;
SortableJS v1.10.2-fvtt1;
SoundBoard by Blitz v1.5.3;
SoundBoard Pack - Blitz Free Pack v1.0.1;
SoundBoard Pack - Community v1.0.1;
Split Journal v1.1;
The Furnace v2.6.1;
Tidy5e Sheet v0.5.20;
Tidy UI - Game Settings v0.1.28;
T is for Target v0.4;
Token Action HUD v1.3.29;
Token Attacher v4.2.1;
Token Info Icons v2.3.5;
Token Mold v2.14.0;
Torch v1.4.3;
Trigger Happy v0.8.22;
Walls Have Ears - Muffled sounds behind walls v0.0.4;
Warp Gate v1.11.2;

Inactive Modules:
----------
About Time v1.0.6;
Actually Private Rolls v0.2;
Adventure Importer / Exporter v0.7.1;
Better NPC Sheet 5e v0.10.1;
Better Roofs v1.3.5;
Blood 'n Guts v0.8.7;
Calendar/Weather v3.1.10;
Chat Portrait v0.1.4;
Combat Ready! v3.2.1;
Deprecated Modules v1.1.6;
Deselection v1.4.4;
dScryb v2.2.0;
Hey, listen! - Chat Notifications v1.2.3;
Hey, Wait! v0.6.3;
Iconizer v2.1.15;
Levels v1.9.9.7;
Lock View v1.4.14;
Loot Sheet NPC 5e v3.4.1;
Michael Ghelfi Studios Audio Pack v1.3;
Mobile Improvements v1.0.3;
Mount Up! v3.0.1;
Multilevel Tokens v1.5.2;
Parallaxia v0.3.6;
Roofs &amp; Overhead Tiles v0.2.1;
Scenery - Background Image Variation Manager v0.1.4;
Show Drag Distance v2.2.3;
Simple Calendar vv1.3.75;
Token Audio v1.0.2;
Token Auras v1.15;
Tokenizer v3.5.0;
Token Magic FX v0.5.2.1;
TouchVTT v1.8.1;
Turn Alert v2.0.0;
Turn Marker v2.8.2;
Vehicles and Mechanisms v0.4.0;
Wall Height v3.6.0.1;
Weather Control v4.1.3;

**Comment by maxphoenix01** [comment]
I made a screen recording of the issue as well.

https://user-images.githubusercontent.com/85245641/147416113-f74ede12-7f04-4b23-805c-62fd29aba38c.MP4

**Comment by maxphoenix01** [comment]
I made a screen recording of the issue as well. I'm not sure what could be causing this problem. I would appreciate your help. Thank you!

https://user-images.githubusercontent.com/85245641/147416154-9f0d0578-5e3f-4e07-8d5d-59eb69fb2291.MP4

**Comment by jofrepouplana** [comment]
The same happens to me since I instaled Midi QoL. I think somehow this module duplicates the order to play the sound but I don't find the way to disable this option.

**Comment by jofrepouplana** [comment]
Uploading VID_20220217_120148825.mp4…

**Comment by eclarke12** [comment]
@maxphoenix01 Is this still occurring?

**Comment by maxphoenix01** [comment]
> @maxphoenix01 Is this still occurring?

Yes it is still occurring with the recent updates to foundry and midi qol.
